src/near/schema.ts

```diff
@@ -4,42 +4,64 @@ import { readCustomSection } from "wasm-walrus-tools"
 import { ContractCodeView } from "near-api-js/lib/providers/provider"
 import { JSONSchema7 } from "json-schema"
 
+
 export async function fetchSchema(contract: string, near: naj.Near): Promise<JSONSchema7> {
   // TODO handle either HTTP endpoint or IPFS hash
-  const url = await fetchJsonAddress(contract, near)
+  const url_or_data = await fetchJsonAddressOrData(contract, near)
```

Comment:
you're working with TS now 😉 

```suggestion
  const urlOrData = await fetchJsonAddressOrData(contract, near)
```

Reply:
too much rust...